### PR TITLE
mysql-connector-c

### PR DESCRIPTION
**Pull request by druvus**
Adds a recipe for mysql-connector-c.

### CODE REVIEW

File: recipes/mysql-connector-c/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+{% set version = "6.1.6" %}
+{% set sha256 = "2222433012c415871958b61bc4f3683e1ebe77e3389f698b267058c12533ea78" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://ftp.iij.ad.jp/pub/db/mysql/Downloads/Connector-C/{{ name|lower }}-{{ version }}-src.tar.gz
```
Comment (synapticarbors) [inline recipes/mysql-connector-c/meta.yaml]: What is this ftp server? I see it's the same url as in the bioconda reciple, but is it owned by MySQL? If not, what is its origin? How is it related to the source tarballs available at:

https://dev.mysql.com/downloads/connector/c/

**Comment by synapticarbors** [comment]
@druvus -- also not sure if you've seen the following pull request for mysqlclient/mysqldb. We could use some input on it:

https://github.com/conda-forge/staged-recipes/pull/1008

Were you planning on using this recipe as a prerequisite for building the python interfaces?

File: recipes/mysql-connector-c/meta.yaml (hunk 1 above)
Comment (druvus) [inline recipes/mysql-connector-c/meta.yaml]: My fault. It is the same file and I was updating the url when I was looking at win source version but forgot to add it. I will update the url and also take another look at the windows version.

**Comment by druvus** [comment]
Yes, I need it as a prerequisite for the python interface so I will take a closer at the PR once the c connecter is working. I got the original mysqldb (mysqldb) recipe working locally on the linux image.

**Comment by druvus** [comment]
I would probably need some help with the win build as I don't have a windows computer.

File: recipes/mysql-connector-c/build.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+mkdir build
+cd build
+
+cmake  -DCMAKE_INSTALL_PREFIX=$PREFIX ..
+
+make
```
Comment (scopatz) [inline recipes/mysql-connector-c/build.sh]: Is there a `make check` you can add?

Comment (druvus) [inline recipes/mysql-connector-c/build.sh]: Unfortunately not.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**